VenafiPS/Public/Get-TppIdentity.ps1

```diff
@@ -121,6 +137,14 @@ function Get-TppIdentity {
                         $response | Add-Member @{ 'Associated' = $associated.Identities }
                     }
 
+                    if (($response.IsGroup) -and ($IncludeMembers))  {
+                        $assocParams = $params.Clone()                       
+                        $assocParams.UriLeaf = 'Identity/GetMembers'
+                        $assocParams.Body.Add("ResolveNested","1");
```

Comment:
I refer the explicit syntax since it emphasizes that that  is a new key, as opposed to simply updating an existing one.  Coding wise I think that makes it easier to read .  I can go either way though.

Reply:
Typically it's best to follow the convention of the existing code so it's all the same.